DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/CourseDelegates/_DelegateProgressSummary.cshtml

```diff
@@ -0,0 +1,165 @@
+        Edit<span class="nhsuk-u-visually-hidden"> completion date</span>
+      </a>
+    </dd>
+  </div>
+
+  <div class="nhsuk-summary-list__row">
+    <dt class="nhsuk-summary-list__key">
+      Evaluation date
+    </dt>
+    <partial name="_SummaryFieldValue" model="Model.Evaluated?.ToString(DateHelper.StandardDateAndTimeFormat)" />
+    <dd class="nhsuk-summary-list__actions" />
+  </div>
+
+  <div class="nhsuk-summary-list__row">
+    <dt class="nhsuk-summary-list__key">
+      Removed date
+    </dt>
+    <partial name="_SummaryFieldValue" model="Model.RemovedDate?.ToString(DateHelper.StandardDateFormat)" />
+    <dd class="nhsuk-summary-list__actions" />
+  </div>
+
+  <div class="nhsuk-summary-list__row">
+    <dt class="nhsuk-summary-list__key">
+      Enrolment method
+    </dt>
+    <dd class="nhsuk-summary-list__value">
+      @Model.EnrolmentMethod
+    </dd>
+    <dd class="nhsuk-summary-list__actions" />
+  </div>
+
+  <div class="nhsuk-summary-list__row">
+    <dt class="nhsuk-summary-list__key">
+      Logins
+    </dt>
+    <dd class="nhsuk-summary-list__value">
+      @Model.LoginCount
+    </dd>
+    <dd class="nhsuk-summary-list__actions" />
+  </div>
+
+  <div class="nhsuk-summary-list__row">
+    <dt class="nhsuk-summary-list__key">
+      Learning time (mins)
+    </dt>
+    <dd class="nhsuk-summary-list__value">
+      @Model.LearningTime
+    </dd>
+    <dd class="nhsuk-summary-list__actions" />
+  </div>
+
+  <div class="nhsuk-summary-list__row">
+    <dt class="nhsuk-summary-list__key">
+      Diagnostic score
+    </dt>
+    <partial name="_SummaryFieldValue" model="Model.DiagnosticScore?.ToString()" />
+    <dd class="nhsuk-summary-list__actions" />
+  </div>
+
+  @if (Model.IsAssessed) {
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Assessments passed
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @Model.AttemptsPassed
+      </dd>
+      <dd class="nhsuk-summary-list__actions" />
+    </div>
+
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Assessment attempts
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @Model.AllAttempts
+      </dd>
+      <dd class="nhsuk-summary-list__actions" />
+    </div>
+
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        Pass rate
+      </dt>
+      <dd class="nhsuk-summary-list__value">
+        @Model.PassRate%
+      </dd>
+      <dd class="nhsuk-summary-list__actions" />
+    </div>
+  }
+
+  @foreach (var prompt in Model.CustomFields) {
+    <div class="nhsuk-summary-list__row">
+      <dt class="nhsuk-summary-list__key">
+        @prompt.CustomPrompt
+      </dt>
+      <partial name="_SummaryFieldValue" model="prompt.Answer" />
+      <dd class="nhsuk-summary-list__actions">
+        <a href="#">
+          Edit<span class="nhsuk-u-visually-hidden"> @prompt.CustomPrompt</span>

```

Comment:
What is this hidden span for?

Reply:
Oh is it for accessibility?

Reply:
It is indeed for accessibility, it's part of the NHS component. It's just so the screen reader announces what you are going to edit by clicking the link, rather than just announcing "Edit" which can be a little confusing at times.

Reply:
Good to know, thanks!